docs/maintainers/issue-management.md

```diff
@@ -4,7 +4,7 @@
 
 If you believe that there is a security vulnerability in PowerShell Core,
 it **must** be reported to [secure@microsoft.com](https://technet.microsoft.com/security/ff852094.aspx) to allow for [Coordinated Vulnerability Disclosure](https://technet.microsoft.com/security/dn467923).
-**Only** file an issue, if secure@microsoft.com has confirmed filing an issue is appropriate.
+**Only** file an issue, if [secure@microsoft.com][https://technet.microsoft.com/security/ff852094.aspx] has confirmed filing an issue is appropriate.
```

Comment:
The URL should be https://www.microsoft.com/en-us/msrc/faqs-report-an-issue?rtc=1

Reply:
Fixed.